projects/canopy-test-app/src/app/app.component.html

```diff
@@ -13,7 +13,7 @@
                   Home
                 </a>
               </lg-breadcrumb-item>
-              <lg-breadcrumb-item>
+              <lg-breadcrumb-item showOnSmScreens="true">
```

Comment:
Just in terms of naming convention we recently adopted this approach for a table property with the intention of keeping the same convention on stacked radio buttons (cc @elenagarrone 😁)...

```
showColumnsAt | Sets the minimum screen width from which the column layout is displayed. Accepts sm, md or lg
```
I wondered wether the same convention could map to this?

e.g.
```
showItemAt | Sets the minimum screen width that the breadcrumb item is displayed. Accepts sm, md or lg
```

Reply:
I initially set out to follow the 'showAt' pattern the table introduced but I pivoted for a few reasons. One was to focus on simplifying what we already had whilst weeding out what was redundant. The second reason was I wasn't sure if it would be overkill (as the original breadcrumb UX was very much focussed on small screen verses non small screen behaviour). That being said if there is an appetite for having a higher degree of control and avoiding further breaking changes I would be happy to look into implementing this

Reply:
Maybe seek a second opinion, I'm mostly coming from a DevEx point point of view, for consistency on the properties. The extra control and breaking change avoidance would be a bonus.

Reply:
I would advocate using the `showItemAt` approach, I think the consistency and flexibility would help us in the long run. It doesn't feel much like YAGNI, because it's achieving the same result in a more flexible way and it's creating consistently too. Not having a breaking change is also a nice to have! 

In terms of the original UX guidelines, perhaps we can address that clearly in the notes - maybe a couple of lines explaining what the guidelines are and that to achieve this, you should use `showItemAt="md"`, with an a code example. Maybe something like this: